app.js

```diff
@@ -0,0 +1,56 @@
+// declare variables we need
+const scoreInput = document.querySelector("#score");
+const diceBox = document.querySelector("#dice-box");
+const rollBtn = document.querySelector("#roll-button");
+const playerStatus = document.querySelector("#player-status");
+const diceImg = document.querySelector("#dice-img");
```

Comment:
Good use of const variables, if they don't need to be changed, don't allow them to be :)